cita-network/src/p2p_protocol/transfer.rs

```diff
@@ -29,6 +29,7 @@ use tentacle::{
 use tokio::codec::length_delimited::LengthDelimitedCodec;
 
 pub const TRANSFER_PROTOCOL_ID: ProtocolId = 1;
+pub const MAX_FRAME_LENGTH: usize = 64 * 1024 * 1204;
```

Comment:
why?: `64 * 1024 * 1204`

Reply:
Let MAX_FRAME_LENGTH = 512M for now. see #426